File: Online1/B/q.cpp
a=int(input())
b=0
while a>=60:
    b+=1
    a-=60
if b<10 and a>9:
    print("0",end='')
    print(b,end='')
    print(":",end='')
    print(a)
elif b<10 and a<10:
    print("0",end='')
    print(b,end='')
    print(":",end='')
    print("0",end='')
    print(a)
else:
    print(b,end='')
    print(":",end='')
    print(a)

File: Online1/B/q.py
n=int (input())
p=n//60
d=n%60
if d<10 and p<10:
    print('0'+str(p)+':'+'0'+str(d))
elif p<10:
    print('0'+str(p)+":"+str(d))
elif d<10:
    print(str(p)+":"+'0'+str(d))
elif p>10 and d>10:
    print(str(p)+':'+str(d))
